src/styles/card.css

```diff
@@ -0,0 +1,105 @@
+/*  CARD BLOCK:  */
+
+.card {
+  padding: 32px 0 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+}
+
+.card__info {
+  display: flex;
+  flex-direction: column;
+  margin: 0 16px;
```

Comment:
Strange solution. It's a fixed distance between `card` borders and it's content, so padding for card looks like a good solution here, you also set vertical paddings to container, so why not set horizontal too 🙂